Crash report: just uploading a picture

### Steps to reproduce
I don't know how reproducible it is, but let me describe what I did:
1. take a picture
2. choose 'share with -> Nextcloud'
3. navigate to another folder and share

### Expected behaviour
- pic gets uploaded

### Actual behaviour
Nopidopie, nothing happened, so I opened the app - which did show me a crash (see below).

after starting the app again, it did start to upload it.

### Can you reproduce this problem on https://try.nextcloud.com?
am using cloud.nextcloud.com

### Environment data
Android version: Samsung S10e - I think it's android 10

Nextcloud app version: beta

Nextcloud server version: cloud.nc.c, latest

Reverse proxy: nope

### Logs
I have only the android log:

```
************ CAUSE OF ERROR ************

java.lang.RuntimeException: Unable to start activity ComponentInfo{com.nextcloud.client/com.owncloud.android.ui.activity.FileDisplayActivity}: java.lang.NullPointerException: Attempt to read from field 'androidx.viewpager.widget.ViewPager com.owncloud.android.databinding.FileDetailsFragmentBinding.pager' on a null object reference
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3827)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4003)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2317)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:246)
	at android.app.ActivityThread.main(ActivityThread.java:8595)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:602)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)
Caused by: java.lang.NullPointerException: Attempt to read from field 'androidx.viewpager.widget.ViewPager com.owncloud.android.databinding.FileDetailsFragmentBinding.pager' on a null object reference
	at com.owncloud.android.ui.fragment.FileDetailFragment.getFileDetailSharingFragment(FileDetailFragment.java:156)
	at com.owncloud.android.ui.activity.FileActivity.getShareFileFragment(FileActivity.java:855)
	at com.owncloud.android.ui.activity.FileActivity.getFile(FileActivity.java:297)
	at com.owncloud.android.ui.activity.FileDisplayActivity.updateActionBarTitleAndHomeButton(FileDisplayActivity.java:1635)
	at com.owncloud.android.ui.activity.ToolbarActivity.setupToolbar(ToolbarActivity.java:103)
	at com.owncloud.android.ui.activity.ToolbarActivity.setupHomeSearchToolbarWithSortAndListButtons(ToolbarActivity.java:127)
	at com.owncloud.android.ui.activity.FileDisplayActivity.onCreate(FileDisplayActivity.java:266)
	at android.app.Activity.performCreate(Activity.java:8207)
	at android.app.Activity.performCreate(Activity.java:8191)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1309)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3800)
	... 11 more

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30170151
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: samsung
Device: beyond0
Model: SM-G970F
Id: RP1A.200720.012
Product: beyond0lteeea

************ FIRMWARE ************
SDK: 30
Release: 11
Incremental: G970FXXSCFUH5
```

--- a/src/main/java/com/owncloud/android/ui/fragment/FileDetailFragment.java
+++ b/src/main/java/com/owncloud/android/ui/fragment/FileDetailFragment.java
@@ -169,6 +169,9 @@ public class FileDetailFragment extends FileFragment implements OnClickListener,
      * @return reference to the {@link FileDetailSharingFragment}
      */
     public FileDetailSharingFragment getFileDetailSharingFragment() {
+        if (binding == null) {
+            return null;
+        }
         return ((FileDetailTabAdapter) binding.pager.getAdapter()).getFileDetailSharingFragment();
     }
 

